### ISSUE TITLE
ISBAT The editor has encountered an unexpected error on WP 5.5.3

### ISSUE THREAD

**Issue by gdandrija** [godaddy-wordpress/coblocks#1849]
### Describe the bug:
Editing or creating pages using the latest version of CoBlocks 2.8.2 and WordPress version 5.5.3 results in an error:

TypeError: r is not a function
    at http://192.168.0.28/wp-includes/js/dist/components.min.js?ver=2152e06558875aa948807b6ffd5da769:7:394727
    at Vb (http://192.168.0.28/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.9.0:104:431)
    at Xi (http://192.168.0.28/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.9.0:151:136)
    at unstable_runWithPriority (http://192.168.0.28/wp-includes/js/dist/vendor/react.min.js?ver=16.9.0:26:340)
    at Ma (http://192.168.0.28/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.9.0:52:280)
    at Yb (http://192.168.0.28/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.9.0:150:420)
    at Ae (http://192.168.0.28/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.9.0:118:178)
    at xi (http://192.168.0.28/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.9.0:39:146)

On WordPress 5.6.2 issue cannot be replicated.

### To reproduce:

1. Go to Pages > All pages
2. Click on Add new or edit any of the existing pages
3. See an error instead of a newly created page
-->

### Expected behavior:
To be able to edit and create pages as usual

### Screenshots:
https://i.imgur.com/C727D0D.png

### Isolating the problem:

- [x ] This bug happens with no other plugins activated
- [x ] This bug happens with a default WordPress theme active
- [x ] This bug happens **without** the Gutenberg plugin active
- [x ] I can reproduce this bug consistently using the steps above

### WordPress version:
5.5.3

### Gutenberg version:
/

Initially submitted on WordPress.org:
https://wordpress.org/support/topic/cant-edit-or-create-pages-error-still-present-in-v2-8-2/

**Comment by snovosel** [comment]
@gdandrija can you try updating to the latest version of CoBlocks ?

**Comment by hvianna** [comment]
Still same issue with CoBlocks 2.9.1 and WP 5.5.3

**Comment by Creatiff** [comment]
Wow!  I am having the same error message, and can't seem to do anything about it!

Before this, when I tried to edit a page I kept receiving a weird edit error massage. 

![Editing-error-message](https://user-images.githubusercontent.com/62785540/115979698-902a9100-a544-11eb-8115-9baab0219c9a.PNG)

I called GoDaddy, because I thought my new hosting wasn't properly configured.  But it was.  One of the techs had to add the following code to .htaccess file:

SubstituteMaxLineLength 10M

Now it seemed to work, until yesterday, when I got the same error message shown above.

![Editor error message_Apr-24-2021](https://user-images.githubusercontent.com/62785540/115979538-b8fe5680-a543-11eb-868d-c12602843e2b.PNG)

I am so disappointed in this new CoBlocks editor, I wish someone would fix it for good; otherwise, just throw it in the garbage!  Sorry to say that, I know its developer had worked hard on it, but if you do something, do it good, without sooooooooo many bugs.  It's very bad!  It does the same with any theme.  I had to re-do 2 of my clients' sites, the whole website several times.  I even re-installed the whole WordPress app a few times, but continues to have the same problem.  Very stressful, because I love Gutenberg blocks.  But since I am working on a client's site, it's very stressful, because the client might say that I don't know what I'm doing.  I am just a designer, not a developer, so, I now nothing about the codes.  But I never had the same problem with WordPress' old editor!

@hvianna, have you resolved your issue?
